diff -ru gettext-0.19.5.1.orig/gettext-tools/src/Makefile.am gettext-0.19.5.1/gettext-tools/src/Makefile.am
--- gettext-0.19.5.1.orig/gettext-tools/src/Makefile.am	2015-06-23 08:46:19.000000000 +0200
+++ gettext-0.19.5.1/gettext-tools/src/Makefile.am	2015-07-21 12:50:26.000000000 +0200
@@ -333,6 +333,7 @@
 recode_sr_latin_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(bindir)\"
 hostname_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(pkglibdir)\"
 urlget_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(pkglibdir)\"
+cldr_plurals_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(pkglibdir)\"
 if RELOCATABLE_VIA_LD
 msgcmp_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(bindir)`
 msgfmt_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(bindir)`
@@ -352,6 +353,7 @@
 recode_sr_latin_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(bindir)`
 hostname_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(pkglibdir)`
 urlget_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(pkglibdir)`
+cldr_plurals_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(pkglibdir)`
 endif
 
 # Linking with C++ libraries is needed _only_ on mingw and Cygwin.
diff -ru gettext-0.19.5.1.orig/gettext-tools/src/Makefile.in gettext-0.19.5.1/gettext-tools/src/Makefile.in
--- gettext-0.19.5.1.orig/gettext-tools/src/Makefile.in	2015-07-15 03:32:29.000000000 +0200
+++ gettext-0.19.5.1/gettext-tools/src/Makefile.in	2015-07-21 12:50:35.000000000 +0200
@@ -486,8 +486,9 @@
 	libgettextsrc_la-read-desktop.lo $(am__objects_3)
 libgettextsrc_la_OBJECTS = $(am_libgettextsrc_la_OBJECTS)
 PROGRAMS = $(bin_PROGRAMS) $(noinst_PROGRAMS)
-am_cldr_plurals_OBJECTS = cldr-plural.$(OBJEXT) \
-	cldr-plural-exp.$(OBJEXT) cldr-plurals.$(OBJEXT)
+am_cldr_plurals_OBJECTS = cldr_plurals-cldr-plural.$(OBJEXT) \
+	cldr_plurals-cldr-plural-exp.$(OBJEXT) \
+	cldr_plurals-cldr-plurals.$(OBJEXT)
 cldr_plurals_OBJECTS = $(am_cldr_plurals_OBJECTS)
 am__DEPENDENCIES_1 =
 @WOE32_TRUE@am__DEPENDENCIES_2 = gettext.res
@@ -498,6 +499,9 @@
 am__v_lt_ = $(am__v_lt_@AM_DEFAULT_V@)
 am__v_lt_0 = --silent
 am__v_lt_1 = 
+cldr_plurals_LINK = $(LIBTOOL) $(AM_V_lt) --tag=CC $(AM_LIBTOOLFLAGS) \
+	$(LIBTOOLFLAGS) --mode=link $(CCLD) $(AM_CFLAGS) $(CFLAGS) \
+	$(cldr_plurals_LDFLAGS) $(LDFLAGS) -o $@
 am_hostname_OBJECTS = hostname-hostname.$(OBJEXT)
 hostname_OBJECTS = $(am_hostname_OBJECTS)
 hostname_LDADD = $(LDADD)
@@ -2110,6 +2114,7 @@
 recode_sr_latin_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(bindir)\"
 hostname_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(pkglibdir)\"
 urlget_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(pkglibdir)\"
+cldr_plurals_CPPFLAGS = $(AM_CPPFLAGS) -DINSTALLDIR=\"$(pkglibdir)\"
 @RELOCATABLE_VIA_LD_TRUE@msgcmp_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(bindir)`
 @RELOCATABLE_VIA_LD_TRUE@msgfmt_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(bindir)`
 @RELOCATABLE_VIA_LD_TRUE@msgmerge_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(bindir)`
@@ -2128,6 +2133,7 @@
 @RELOCATABLE_VIA_LD_TRUE@recode_sr_latin_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(bindir)`
 @RELOCATABLE_VIA_LD_TRUE@hostname_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(pkglibdir)`
 @RELOCATABLE_VIA_LD_TRUE@urlget_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(pkglibdir)`
+@RELOCATABLE_VIA_LD_TRUE@cldr_plurals_LDFLAGS = `$(RELOCATABLE_LDFLAGS) $(pkglibdir)`
 
 # Linking with C++ libraries is needed _only_ on mingw and Cygwin.
 @WOE32DLL_FALSE@libgettextsrc_la_LINK = $(LIBTOOL) $(AM_V_lt) --tag=CC \
@@ -2417,7 +2423,7 @@
 
 cldr-plurals$(EXEEXT): $(cldr_plurals_OBJECTS) $(cldr_plurals_DEPENDENCIES) $(EXTRA_cldr_plurals_DEPENDENCIES) 
 	@rm -f cldr-plurals$(EXEEXT)
-	$(AM_V_CCLD)$(LINK) $(cldr_plurals_OBJECTS) $(cldr_plurals_LDADD) $(LIBS)
+	$(AM_V_CCLD)$(cldr_plurals_LINK) $(cldr_plurals_OBJECTS) $(cldr_plurals_LDADD) $(LIBS)
 
 hostname$(EXEEXT): $(hostname_OBJECTS) $(hostname_DEPENDENCIES) $(EXTRA_hostname_DEPENDENCIES) 
 	@rm -f hostname$(EXEEXT)
@@ -2721,6 +2727,24 @@
 ../woe32dll/libgettextsrc_la-gettextsrc-exports.lo: ../woe32dll/gettextsrc-exports.c
 	$(AM_V_CC)$(LIBTOOL) $(AM_V_lt) --tag=CC $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(libgettextsrc_la_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) -c -o ../woe32dll/libgettextsrc_la-gettextsrc-exports.lo `test -f '../woe32dll/gettextsrc-exports.c' || echo '$(srcdir)/'`../woe32dll/gettextsrc-exports.c
 
+cldr_plurals-cldr-plural.o: cldr-plural.c
+	$(AM_V_CC)$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(cldr_plurals_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) -c -o cldr_plurals-cldr-plural.o `test -f 'cldr-plural.c' || echo '$(srcdir)/'`cldr-plural.c
+
+cldr_plurals-cldr-plural.obj: cldr-plural.c
+	$(AM_V_CC)$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(cldr_plurals_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) -c -o cldr_plurals-cldr-plural.obj `if test -f 'cldr-plural.c'; then $(CYGPATH_W) 'cldr-plural.c'; else $(CYGPATH_W) '$(srcdir)/cldr-plural.c'; fi`
+
+cldr_plurals-cldr-plural-exp.o: cldr-plural-exp.c
+	$(AM_V_CC)$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(cldr_plurals_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) -c -o cldr_plurals-cldr-plural-exp.o `test -f 'cldr-plural-exp.c' || echo '$(srcdir)/'`cldr-plural-exp.c
+
+cldr_plurals-cldr-plural-exp.obj: cldr-plural-exp.c
+	$(AM_V_CC)$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(cldr_plurals_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) -c -o cldr_plurals-cldr-plural-exp.obj `if test -f 'cldr-plural-exp.c'; then $(CYGPATH_W) 'cldr-plural-exp.c'; else $(CYGPATH_W) '$(srcdir)/cldr-plural-exp.c'; fi`
+
+cldr_plurals-cldr-plurals.o: cldr-plurals.c
+	$(AM_V_CC)$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(cldr_plurals_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) -c -o cldr_plurals-cldr-plurals.o `test -f 'cldr-plurals.c' || echo '$(srcdir)/'`cldr-plurals.c
+
+cldr_plurals-cldr-plurals.obj: cldr-plurals.c
+	$(AM_V_CC)$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(cldr_plurals_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) -c -o cldr_plurals-cldr-plurals.obj `if test -f 'cldr-plurals.c'; then $(CYGPATH_W) 'cldr-plurals.c'; else $(CYGPATH_W) '$(srcdir)/cldr-plurals.c'; fi`
+
 hostname-hostname.o: hostname.c
 	$(AM_V_CC)$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(hostname_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) -c -o hostname-hostname.o `test -f 'hostname.c' || echo '$(srcdir)/'`hostname.c
 
