diff -ru gettext-0.22.orig/gettext-tools/src/msgexec.c gettext-0.22/gettext-tools/src/msgexec.c
--- gettext-0.22.orig/gettext-tools/src/msgexec.c	2023-06-17 12:58:50.000000000 +0200
+++ gettext-0.22/gettext-tools/src/msgexec.c	2023-07-07 19:15:01.000000000 +0200
@@ -325,6 +325,8 @@
 
 #ifdef EINTR
 
+#undef close
+
 /* EINTR handling for close().
    These functions can return -1/EINTR even though we don't have any
    signal handlers set up, namely when we get interrupted via SIGSTOP.  */
Only in gettext-0.22/gettext-tools/src: msggrep.c.orig
diff -ru gettext-0.22.orig/gettext-tools/src/urlget.c gettext-0.22/gettext-tools/src/urlget.c
--- gettext-0.22.orig/gettext-tools/src/urlget.c	2023-06-17 12:58:50.000000000 +0200
+++ gettext-0.22/gettext-tools/src/urlget.c	2023-07-07 19:15:01.000000000 +0200
@@ -49,6 +49,7 @@
 # define STDOUT_FILENO 1
 #endif
 
+#undef close
 
 /* Only high-level toolkits, written in languages with exception handling,
    have an URL datatype and operations to fetch an URL's contents.  Such
diff -ru gettext-0.22.orig/gettext-tools/src/write-catalog.c gettext-0.22/gettext-tools/src/write-catalog.c
--- gettext-0.22.orig/gettext-tools/src/write-catalog.c	2020-07-17 23:45:27.000000000 +0200
+++ gettext-0.22/gettext-tools/src/write-catalog.c	2023-07-07 19:15:01.000000000 +0200
@@ -59,6 +59,8 @@
 
 #endif
 
+#undef close
+
 
 /* =========== Some parameters for use by 'msgdomain_list_print'. ========== */
 
