diff --git a/gettext-tools/src/msgexec.c b/gettext-tools/src/msgexec.c
index fb7e2c8a0..bcb6ae354 100644
--- a/gettext-tools/src/msgexec.c
+++ b/gettext-tools/src/msgexec.c
@@ -325,6 +325,8 @@ or by email to <%s>.\n"),
 
 #ifdef EINTR
 
+#undef close
+
 /* EINTR handling for close().
    These functions can return -1/EINTR even though we don't have any
    signal handlers set up, namely when we get interrupted via SIGSTOP.  */
diff --git a/gettext-tools/src/msggrep.c b/gettext-tools/src/msggrep.c
index e3b7a9ca4..a5fdbaa04 100644
--- a/gettext-tools/src/msggrep.c
+++ b/gettext-tools/src/msggrep.c
@@ -658,6 +658,8 @@ filename_list_match (const string_list_ty *slp, const char *filename)
 
 #ifdef EINTR
 
+#undef close
+
 /* EINTR handling for close().
    These functions can return -1/EINTR even though we don't have any
    signal handlers set up, namely when we get interrupted via SIGSTOP.  */
diff --git a/gettext-tools/src/urlget.c b/gettext-tools/src/urlget.c
index ca3df36e7..0f2df9816 100644
--- a/gettext-tools/src/urlget.c
+++ b/gettext-tools/src/urlget.c
@@ -50,6 +50,7 @@
 # define STDOUT_FILENO 1
 #endif
 
+#undef close
 
 /* Only high-level toolkits, written in languages with exception handling,
    have an URL datatype and operations to fetch an URL's contents.  Such
diff --git a/gettext-tools/src/write-catalog.c b/gettext-tools/src/write-catalog.c
index c25916d1b..479a7efdb 100644
--- a/gettext-tools/src/write-catalog.c
+++ b/gettext-tools/src/write-catalog.c
@@ -59,6 +59,8 @@
 
 #endif
 
+#undef close
+
 
 /* =========== Some parameters for use by 'msgdomain_list_print'. ========== */
 
