diff --git gettext-runtime/intl/printf.c gettext-runtime/intl/printf.c
index 0369e87..b4a9c29 100644
--- gettext-runtime/intl/printf.c
+++ gettext-runtime/intl/printf.c
@@ -41,6 +41,10 @@ char *alloca ();
 # endif
 #endif
 
+#if defined(_WIN32) && !defined(HAVE_DECL__SNWPRINTF)
+# define HAVE_DECL__SNWPRINTF 1
+#endif
+
 #include <stdio.h>
 
 #if !HAVE_POSIX_PRINTF
diff --git gettext-runtime/intl/vasnprintf.c gettext-runtime/intl/vasnprintf.c
index ecccd86..8b6e22b 100644
--- gettext-runtime/intl/vasnprintf.c
+++ gettext-runtime/intl/vasnprintf.c
@@ -50,6 +50,10 @@
 # define _GNU_SOURCE    1
 #endif
 
+#if defined(_WIN32) && !defined(HAVE_DECL__SNWPRINTF)
+# define HAVE_DECL__SNWPRINTF 1
+#endif
+
 #ifndef VASNPRINTF
 # include <config.h>
 #endif
diff --git gettext-runtime/libasprintf/vasnprintf.c gettext-runtime/libasprintf/vasnprintf.c
index ecccd86..8b6e22b 100644
--- gettext-runtime/libasprintf/vasnprintf.c
+++ gettext-runtime/libasprintf/vasnprintf.c
@@ -50,6 +50,10 @@
 # define _GNU_SOURCE    1
 #endif
 
+#if defined(_WIN32) && !defined(HAVE_DECL__SNWPRINTF)
+# define HAVE_DECL__SNWPRINTF 1
+#endif
+
 #ifndef VASNPRINTF
 # include <config.h>
 #endif
diff --git gettext-tools/gnulib-lib/vasnprintf.c gettext-tools/gnulib-lib/vasnprintf.c
index 5267b1b..44919de 100644
--- gettext-tools/gnulib-lib/vasnprintf.c
+++ gettext-tools/gnulib-lib/vasnprintf.c
@@ -50,6 +50,10 @@
 # define _GNU_SOURCE    1
 #endif
 
+#if defined(_WIN32) && !defined(HAVE_DECL__SNWPRINTF)
+# define HAVE_DECL__SNWPRINTF 1
+#endif
+
 #ifndef VASNPRINTF
 # include <config.h>
 #endif
diff --git gettext-tools/libgettextpo/vasnprintf.c gettext-tools/libgettextpo/vasnprintf.c
index 5267b1b..44919de 100644
--- gettext-tools/libgettextpo/vasnprintf.c
+++ gettext-tools/libgettextpo/vasnprintf.c
@@ -50,6 +50,10 @@
 # define _GNU_SOURCE    1
 #endif
 
+#if defined(_WIN32) && !defined(HAVE_DECL__SNWPRINTF)
+# define HAVE_DECL__SNWPRINTF 1
+#endif
+
 #ifndef VASNPRINTF
 # include <config.h>
 #endif
